src/main/java/roboy/ros/RoslessCalls.java

```diff
@@ -0,0 +1,15 @@
+package roboy.ros;
+
+import org.roboy.memory.ros.ServiceReplacement;
```

Comment:
also, i do not see you cleaned up the RosMainNode from the deprecated functions (ROS service calls to memory). if you are not sure about completely deleting them, please mark with the java tag deprecated in the code

Reply:
3547307